bin/XMLToSARIF.sh

```diff
@@ -79,47 +83,46 @@ done
 
 # Validate options value
 if [ -z "$xml_report_path" ]; then
-    echo "Error: inputXmlReport is required."
+    echo "Error: \"-i\" or \"--inputXmlReport\" is required."
     print_usage
     exit 1
 elif [ ! -f "$xml_report_path" ]; then
-    echo "Error: inputXmlReport file not found."
+    echo "Error: \"$xml_report_path\" not found."
     exit 1
 fi
 
 # Set default SARIF report path if not provided
 if [ -z "$sarif_report_path" ]; then
-    sarif_report="${xml_report_path%.*}.sarif"
-else
-    sarif_report="$sarif_report_path"
+    sarif_report_path="${xml_report_path%.*}.sarif"
 fi
 
 # Find Java path
-if [ -z "$java_or_parasoft_tool_root_path" ]; then
-    embedded_java=$(which java)
-    if [ -z "$embedded_java" ]; then
-        echo "Error: Java is not found. Please add java in environment variable or provide javaOrToolHomeDir option."
+if [ -z "$parasoft_tool_or_java_root_path" ]; then
+    java_path=$(which java)
+    if [ -z "$java_path" ]; then
+        echo "Error: Java is not found. Please add Java in environment variable or provide \"-t\" or \"--toolOrJavaHomeDir\" option."
         exit 1;
     fi
-    java_path="$embedded_java"
 else
-     if results=$(getJavaPath "$java_or_parasoft_tool_root_path"); then
+    if results=$(getJavaPath "$parasoft_tool_or_java_root_path"); then
         java_path="$results"
     else
-        echo "ToolHomeDir is incorrect: $java_or_parasoft_tool_root_path. Please check javaOrToolHomeDir value."
+        echo "Error: Tool or Java home directory is incorrect: \"$parasoft_tool_or_java_root_path\". Please check \"-t\" or \"--toolOrJavaHomeDir\" value."
         exit 1;
     fi
 fi
 
 ## 4. Generate SARIF report
 
-command="$java_path -jar $BASE_DIR/lib/saxon/saxon-he-12.2.jar -xsl:$SARIF_XSL_PATH -s:\"$xml_report_path\" -o:\"$sarif_report\" "
+command="$java_path -jar $BASE_DIR/lib/saxon/saxon-he-12.2.jar -xsl:$SARIF_XSL_PATH -s:\"$xml_report_path\" -o:\"$sarif_report_path\" "
```

Comment:
I tried to execute this script in Git Bash and encountered an error due to a space in `java_path`. 
Please fix this issue and double-check if there are other places in the script that need quotes to avoid similar problems.